modules/ROOT/pages/monitoring/metric-manager.adoc

```diff
@@ -2,11 +2,37 @@
 :description: This section describes the Metric manager of Neo4j Ops Manager.
 
 The metrics page offers rich charts and tables with data related to your monitored DBMSs.
-Currently three time windows are supported Live (30 minutes), 24 hours and 3 days worth of data.
+Currently six time windows are supported, from 30 minutes (live) up to 30 days.
+To allow efficient metric storage, metrics data older than 3 days is aggregated to hourly summary.
 
 The table in the *Host Metrics* tab and *Instance Metrics* tab includes static data for our DBMSs, like the CPU cores of the system (physical and logical) and memory.
 
 image::dashboard.png[width=800]
 
 Charts allow for zooming/panning by clicking and specifying the area of interest by dragging the cursor.
-To reset zoom in the chart you can press double click inside the chart.
\ No newline at end of file
+To reset zoom in the chart you can press double click inside the chart.
+Zoom/reset zoom affects all charts displayed on the page to allow investigating different metrics in a specific time range.
+
+== Setting metric thresholds
+
+To inform administrators about noticeable changes in metrics by showing alerts and sending notifications, NOM supports defining metric thresholds directly from the metrics chart.
+This can be accomplished using "Threshold" menu item of the  "..." menu of the metric chart.
+
+In the example below, a threshold of 6.98GB was added in the chart for "Host memory used" metric.
```

Comment:
Might be nicer to have an example with a non-byte unit since that looks a bit nicer. Plan is to improve the size unit input further on.